commons/src/test/java/info/metadude/android/eventfahrplan/commons/extensions/StandardExtensionsTest.kt

```diff
@@ -0,0 +1,26 @@
+package info.metadude.android.eventfahrplan.commons.extensions
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+
+class StandardExtensionsTest {
```

Comment:
You don't test the return value of `onFalse`, but the production code relies on it.

Reply:
:heavy_check_mark: Added in b24d97f6f966386f0d86c8694f30d7f1728c8697.